src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -118,7 +76,7 @@ private String getUploadedFileUrlToImageSource(HttpServletRequest request, Strin
 
     // User submitted form without selecting a file, so we can't get a URL. (devserver)
     if(blobKeys == null || blobKeys.isEmpty()) {
-      return null;
+      return "";
```

Comment:
Looks like a wrong merge?

Reply:
Yeh, sorry i didn't notice it

Reply:
Oh I think it was not a wrong merge! 
If we return null , a normal message without uploading an image will have a unexpected suffix : "null"